RELEASE_NOTES.md

```diff
@@ -0,0 +1,63 @@
+
+For a more detailed look at all changes see the [changelog](https://github.com/cosmos/cosmos-sdk/blob/master/CHANGELOG.md), or alternatively you can look at issues tracked in 
+the [Cosmos SDK v0.40 milestone](https://github.com/cosmos/cosmos-sdk/milestone/25?closed=1) and [IBC 1.0 milestone](https://github.com/cosmos/cosmos-sdk/milestone/21?closed=1), which contain links to all major SDK issues that were tracked in this release.
+
+## Protocol Buffer Migration
+
+Stargate introduces [protocol buffers](https://developers.google.com/protocol-buffers) as the new standard serialization format for blockchain state & wire communication within the Cosmos SDK. Protobuf definitions are organized into packages that mirror Cosmos SDK modules in the new [./proto](https://github.com/cosmos/cosmos-sdk/tree/master/proto) directory of the SDK repo.
+
+For an overview of the SDK's usage of protocol buffers, please look at the following ADRs and meta-issues which tracked the corresponding work:
+- [ADR019 Protobuf State Encoding](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-019-protobuf-state-encoding.md) / [Full Proto Encoding (#5444)](https://github.com/cosmos/cosmos-sdk/issues/5444)
+- [ADR020 Protobuf Transaction Encoding](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-020-protobuf-transaction-encoding.md) / [Proto Any Tx Migration (#6213)](https://github.com/cosmos/cosmos-sdk/issues/6213) 
+- [ADR021 Protobuf Query Encoding](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-021-protobuf-query-encoding.md) / [Query Protobuf Migration (#5921)](https://github.com/cosmos/cosmos-sdk/issues/5921)
+- [ADR023 Protobuf Naming Conventions](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-023-protobuf-naming.md)
+- [ADR027 Deterministic Protobuf Serialization](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-027-deterministic-protobuf-serialization.md)
+
+As a high level summary these represent the following major changes to the SDK:
+- New protocol buffer based encoding for all blockchain state (direct queries to tendermint now return protobuf binary encoded data, as opposed to Amino encoded data)
+- New transaction signing path implemented according to ADR020 above
+- Two new querier APIs (see [#5921](https://github.com/cosmos/cosmos-sdk/issues/5921) for details)
+  - Support for new gRPC based querier services
+  - gRPC Gateway for REST querying corresponding to the new gRPC querier services
+
+**Note:** Existing Amino REST endpoints are all preserved, though they are planned to be deprecated in a future release.
+
+## Inter Blockchain Communication (IBC)
+
+The `x/ibc` module is now available and ready for use. For more details check the the  [`x/ibc/spec`](https://github.com/cosmos/tree/master/x/ibc/spec) directory:
```

Comment:
the spec link is outdated. It should be under `x/ibc/core/spec`

Reply:
It might be useful to link 3 different things here. The [ICS spec](https://github.com/cosmos/ics/tree/master/spec), the core code spec and the [IBC docs](https://github.com/cosmos/cosmos-sdk/tree/master/docs/ibc)?